### ISSUE TITLE
508-defect-4 [SCREENREADER]: Paragraphs should be written as paragraphs, not headings

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19738]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a user, I expect headings to semantically be headings to help me navigate through the page.

## Details

There are two paragraphs on the 526ez page that are written as headings. Semantically, it may not make sense for them to be within the outline structure. Since they are very verbose, it may be frustrating to screen reader users to listen to.

For example:

## Acceptance Criteria

- [ ] Paragraphs use `<p>`

## Steps to Recreate

See screenshot

## Proposed Solution

These paragraphs should be written as `<p>`s instead of headings. This is more semantically appropriate given their content. If the designer wants to bring emphasis to these sections, a different approach can be used to better outline the page. 

For example, instead of:

> ### When you file a disability claim, you’ll have a chance to provide evidence to support your claim. Evidence could include:

You could try:

> ### Providing Evidence
> When you file a disability claim, you’ll have a chance to provide evidence to support your claim. Evidence could include:

## Screenshots or Trace Logs

<img width="1406" alt="Screen Shot 2021-02-10 at 10 33 59 AM" src="https://user-images.githubusercontent.com/14154792/107532286-b9659300-6b8b-11eb-94a0-75273774a94c.png">

**Comment by Mottie** [comment]
@joshkimux Asha just merged in a fix for this, please review the change at your leisure 😸

**Comment by joshkimux** [comment]
<img width="1117" alt="Screen Shot 2021-05-19 at 11 34 31 AM" src="https://user-images.githubusercontent.com/14154792/118841289-304eae00-b896-11eb-91f1-5d869b5246a3.png">

Perfect! Thank you @ashagross 🥳
